src/e2e-test/regression-test/config/testcases-hl7v2.js

```diff
@@ -3,8 +3,13 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-const cases = [
-    { dataFile: 'ADT01-28.hl7', templateFile: 'ORU_R01.hbs' }
-];
+const dataFiles = [ 'ADT01-28.hl7' ];
+const templateFiles = [ 'ORU_R01.hbs' ];
 
-module.exports = cases;
\ No newline at end of file
+module.exports = () => {
```

Comment:
why not export a variable cases here?

Reply:
Because the json objects can be modified if directly exposed. Exposing by invoking a function and create a new copy will prevent data from pollution. (Of course `Object.freeze(data)` is also a solution)